prometheus/monitoring-alert-rules/README.md

```diff
@@ -7,11 +7,10 @@ This example shows how to deploy and view alerting rules in Kyma.
 ## Prerequisites
 
 * Kyma as the target deployment environment.
+* Deployed custom kube-prometheus-stack as described in the [prometheus](../) example.
 
 ## Installation
 
-You need access to the `kyma-system` Namespace to execute the described steps.
-
 ### Add a new alerting rule
 
 1. Create the PrometheusRule resource holding the configuration of your alerting rule. 
```

Comment:
Maybe we should put there a path to the raw version of the file as we did it in prometheus tutorial?